lib/versioned/runner.js

```diff
@@ -99,7 +99,7 @@ function runTests(cb) {
   // TODO: Add tap arguments, such as color.
   process.send({ status: 'running' })
   let args = [testFile]
-  if (process.env.NR_LOADER) {
+  if (process.env.nr_esm_loader) {
     const loaderPath = path.resolve(process.env.NR_LOADER)
```

Comment:
this is the wrong env var now. should we unset this env var after every run so subsequent suites will not inherit?

Reply:
I'd agree that we should unset if we want to be sure.

Reply:
this is out of date. it's been updated